api/v2/storageproviders.go

```diff
@@ -288,35 +300,19 @@ type storageProviderChainInfo struct {
 // @Param        sp  path      string  true  "Filter by storage provider"
 // @Router       /storage-providers/stats/{sp} [get]
 func (s *apiV2) handleGetStorageProviderStats(c echo.Context) error {
-	ctx, span := s.tracer.Start(c.Request().Context(), "handleGetStorageProviderStats")
```

Comment:
Shouldn't this be ` handleGetStorageProviderStats` on v2? Unless we want to use legacy names code-wise.